docs/dev/dev_environment.md

```diff
@@ -24,7 +24,7 @@ Using **Invoke** these configuration options can be overridden using [several me
 
 ### Docker Development Environment
 
-!!! tip
+!!! TIP
```

Comment:
Why are you uppercasing the admonitions? Does it work?

Theme docs define them all lower case https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Reply:
Seems to me better readable. Reverted to align with official recommendation.